spec/features/register_spec.rb

```diff
@@ -0,0 +1,73 @@
+feature 'Users can sign up for the service' do
+
+  it 'with valid and correct entries' do
```

Comment:
word your `it` tests according to the expected behaviour; not just the context.  it _does what_ with valid and correct entries?  Alternatively, use `scenario` instead of `it`.